diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__0f63b49a542346bd91fa0e78bbd2f6d6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__0f63b49a542346bd91fa0e78bbd2f6d6.uda
index 7160d1a..82a36ad 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__0f63b49a542346bd91fa0e78bbd2f6d6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__0f63b49a542346bd91fa0e78bbd2f6d6.uda
@@ -93,5 +93,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__19fbebfdb27f42c5bdb88686c45f40cc.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__19fbebfdb27f42c5bdb88686c45f40cc.uda
index c8daf53..3a4f5bf 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__19fbebfdb27f42c5bdb88686c45f40cc.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__19fbebfdb27f42c5bdb88686c45f40cc.uda
@@ -50,5 +50,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__2cf4d65039ec41ef8bd65085ee9f780a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__2cf4d65039ec41ef8bd65085ee9f780a.uda
index 12b96d8..fd39fe7 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__2cf4d65039ec41ef8bd65085ee9f780a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__2cf4d65039ec41ef8bd65085ee9f780a.uda
@@ -91,5 +91,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__3f35b1574cb747abbedd2e51551b3706.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__3f35b1574cb747abbedd2e51551b3706.uda
index 0dc74aa..82fad13 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__3f35b1574cb747abbedd2e51551b3706.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__3f35b1574cb747abbedd2e51551b3706.uda
@@ -39,5 +39,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__539869ea4f0d4c3b80b1aaa80a1c16e6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__539869ea4f0d4c3b80b1aaa80a1c16e6.uda
index 7358d66..176a61b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__539869ea4f0d4c3b80b1aaa80a1c16e6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__539869ea4f0d4c3b80b1aaa80a1c16e6.uda
@@ -38,5 +38,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__5c3c3536d2804a8f80bddccbe9222e20.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__5c3c3536d2804a8f80bddccbe9222e20.uda
index 99de7aa..ed28da6 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__5c3c3536d2804a8f80bddccbe9222e20.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__5c3c3536d2804a8f80bddccbe9222e20.uda
@@ -67,5 +67,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__7b2f3de2c7ab449288e8c6bb02eaa6a3.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__7b2f3de2c7ab449288e8c6bb02eaa6a3.uda
index 62d41ff..6607219 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__7b2f3de2c7ab449288e8c6bb02eaa6a3.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__7b2f3de2c7ab449288e8c6bb02eaa6a3.uda
@@ -33,5 +33,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__89f686616e424541bf8e029eef0325ab.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__89f686616e424541bf8e029eef0325ab.uda
index 5630793..3f773fe 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__89f686616e424541bf8e029eef0325ab.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__89f686616e424541bf8e029eef0325ab.uda
@@ -77,5 +77,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__9db112c5c2ea441d8bd46daf522aa2b6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__9db112c5c2ea441d8bd46daf522aa2b6.uda
index 65390bd..17b3e64 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__9db112c5c2ea441d8bd46daf522aa2b6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__9db112c5c2ea441d8bd46daf522aa2b6.uda
@@ -55,5 +55,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ab28f7b5eb974f7994ee0af66fb44128.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ab28f7b5eb974f7994ee0af66fb44128.uda
index 107aae3..19bbe36 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ab28f7b5eb974f7994ee0af66fb44128.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ab28f7b5eb974f7994ee0af66fb44128.uda
@@ -76,5 +76,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ac6ebe34fc2b4ee09b26e7a22adbe591.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ac6ebe34fc2b4ee09b26e7a22adbe591.uda
index b2383c3..eb04e84 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ac6ebe34fc2b4ee09b26e7a22adbe591.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__ac6ebe34fc2b4ee09b26e7a22adbe591.uda
@@ -35,5 +35,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b0287f57fb534b809d0beb7c8b277a5a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b0287f57fb534b809d0beb7c8b277a5a.uda
index dbc22ba..4ac3c46 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b0287f57fb534b809d0beb7c8b277a5a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b0287f57fb534b809d0beb7c8b277a5a.uda
@@ -66,5 +66,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e667d8521e3541d1b0eb46db0ec2342c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e667d8521e3541d1b0eb46db0ec2342c.uda
index 7b2bd97..c36cc9e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e667d8521e3541d1b0eb46db0ec2342c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e667d8521e3541d1b0eb46db0ec2342c.uda
@@ -82,5 +82,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__eb9fd2d152584bf8beaba2415b2e365a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__eb9fd2d152584bf8beaba2415b2e365a.uda
index f207ff3..eaa315e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__eb9fd2d152584bf8beaba2415b2e365a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__eb9fd2d152584bf8beaba2415b2e365a.uda
@@ -65,5 +65,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
index b52d849..f72deb2 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
@@ -3,7 +3,7 @@
   "Alias": "site-content-guidelines",
   "Resources": [
     {
-      "Id": "e3fdb522-8579-4041-9397-c0b0d6dd71b9",
+      "Id": "ac97205e-cec9-44dd-a802-8b33caa7f7bb",
       "ResourceTypeId": "brand-voice",
       "Name": "Brand Voice",
       "Description": null,
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
index 14eb810..1df91a4 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
@@ -3,7 +3,7 @@
   "Alias": "how-to-guide-style",
   "Resources": [
     {
-      "Id": "f329af50-6aa0-44f3-9f58-683e711fa7b9",
+      "Id": "0a64fe34-32a6-4a38-aac8-6bb991bf2909",
       "ResourceTypeId": "text",
       "Name": "Structural Template",
       "Description": "Section-by-section structure every generated guide must follow.",
