diff --git a/src/UmbracoProject/UmbracoProject.csproj b/src/UmbracoProject/UmbracoProject.csproj
index 14f2d8c..de15153 100644
--- a/src/UmbracoProject/UmbracoProject.csproj
+++ b/src/UmbracoProject/UmbracoProject.csproj
@@ -91,7 +91,7 @@
     <PackageReference Include="Umbraco.Cms.Search.DeliveryApi" Version="1.0.0" />
     <PackageReference Include="Umbraco.Cms.Search.Provider.Examine" Version="1.0.0-beta.9" />
     <PackageReference Include="Umbraco.Deploy.Cloud" Version="17.2.0" />
-    <PackageReference Include="Umbraco.Forms" Version="17.4.5" />
+    <PackageReference Include="Umbraco.Forms" Version="17.4.6" />
     <PackageReference Include="Umbraco.Forms.Deploy" Version="17.0.0" />
   </ItemGroup>
 
