diff --git a/src/UmbracoProject/Views/Author.cshtml b/src/UmbracoProject/Views/Author.cshtml
index a9435a6..625ae21 100644
--- a/src/UmbracoProject/Views/Author.cshtml
+++ b/src/UmbracoProject/Views/Author.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Author>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Author>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using Umbraco.Extensions
@@ -69,4 +69,4 @@
             }
         </div>
     </main>
-}
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/AuthorList.cshtml b/src/UmbracoProject/Views/AuthorList.cshtml
index 009b75d..bc8ee8b 100644
--- a/src/UmbracoProject/Views/AuthorList.cshtml
+++ b/src/UmbracoProject/Views/AuthorList.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.AuthorList>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.AuthorList>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using UmbracoProject.Helpers
@@ -20,4 +20,4 @@
     Pattern = Model.PageHeadPattern()
 })
 
-@await Html.PartialAsync("~/Views/Partials/authors.cshtml")
+@await Html.PartialAsync("~/Views/Partials/authors.cshtml")
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/Login.cshtml b/src/UmbracoProject/Views/Login.cshtml
index ea0419d..02be09d 100644
--- a/src/UmbracoProject/Views/Login.cshtml
+++ b/src/UmbracoProject/Views/Login.cshtml
@@ -1,4 +1,4 @@
-@using Umbraco.Cms.Web.Common.PublishedModels;
+﻿@using Umbraco.Cms.Web.Common.PublishedModels;
 @using Umbraco.Extensions;
 @inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Login>
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels;
@@ -76,4 +76,4 @@
             </div>
         </div>
     </div>
-</article>
+</article>
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/article.cshtml b/src/UmbracoProject/Views/article.cshtml
index 3fd933b..7a12dfb 100644
--- a/src/UmbracoProject/Views/article.cshtml
+++ b/src/UmbracoProject/Views/article.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Article>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Article>
 
 @using Clean.Core.Models.ViewModels
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
@@ -99,4 +99,4 @@
             }
         </div>
     </section>
-}
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/articleList.cshtml b/src/UmbracoProject/Views/articleList.cshtml
index 0060188..299675a 100644
--- a/src/UmbracoProject/Views/articleList.cshtml
+++ b/src/UmbracoProject/Views/articleList.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.ArticleList>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.ArticleList>
 
 @using Clean.Core.Helpers
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
@@ -72,4 +72,4 @@
     {
         @await Component.InvokeAsync("Pagination", new { totalItems = totalItemCount, url = Model.Url(), pageNumber = pageNumber, pageSize = pageSize })
     }
-</main>
+</main>
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/contact.cshtml b/src/UmbracoProject/Views/contact.cshtml
index e10e49b..e6dc6ca 100644
--- a/src/UmbracoProject/Views/contact.cshtml
+++ b/src/UmbracoProject/Views/contact.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Contact>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Contact>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using UmbracoProject.Helpers
@@ -46,4 +46,4 @@
             }
         </div>
     }
-</main>
+</main>
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/content.cshtml b/src/UmbracoProject/Views/content.cshtml
index 97d0d74..c2f8469 100644
--- a/src/UmbracoProject/Views/content.cshtml
+++ b/src/UmbracoProject/Views/content.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Content>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Content>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using UmbracoProject.Helpers
@@ -64,4 +64,4 @@
             @await Html.PartialAsync("~/Views/Partials/contentSectionRow.cshtml", row)
         }
     }
-}
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/documentation.cshtml b/src/UmbracoProject/Views/documentation.cshtml
index 728efe2..4a0f5e4 100644
--- a/src/UmbracoProject/Views/documentation.cshtml
+++ b/src/UmbracoProject/Views/documentation.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Documentation>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Documentation>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using UmbracoProject.Helpers
@@ -44,4 +44,4 @@
     {
         @await Html.PartialAsync("~/Views/Partials/contentSectionRow.cshtml", row)
     }
-}
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/error.cshtml b/src/UmbracoProject/Views/error.cshtml
index 0f91854..2486bf3 100644
--- a/src/UmbracoProject/Views/error.cshtml
+++ b/src/UmbracoProject/Views/error.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Error>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Error>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 
@@ -35,4 +35,4 @@
 {
     Headline = "Find your way back",
     Actions = actions
-})
+})
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/home.cshtml b/src/UmbracoProject/Views/home.cshtml
index ec045f1..f4c5d62 100644
--- a/src/UmbracoProject/Views/home.cshtml
+++ b/src/UmbracoProject/Views/home.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Home>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Home>
 
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 
@@ -23,4 +23,4 @@
     <section class="wrap-narrow">
         @Html.GetBlockListHtml(Model.ContentRows)
     </section>
-}
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/master.cshtml b/src/UmbracoProject/Views/master.cshtml
index 7b2a7e8..b51c163 100644
--- a/src/UmbracoProject/Views/master.cshtml
+++ b/src/UmbracoProject/Views/master.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage
 
 @using Umbraco.Forms.Web
 
@@ -45,4 +45,4 @@
     <script src="/assets/js/carousel.js" asp-append-version="true"></script>
     @await RenderSectionAsync("Scripts", required: false)
 </body>
-</html>
+</html>
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/search.cshtml b/src/UmbracoProject/Views/search.cshtml
index f513184..08cbc21 100644
--- a/src/UmbracoProject/Views/search.cshtml
+++ b/src/UmbracoProject/Views/search.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Search>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Search>
 
 @using System.Net
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
@@ -84,4 +84,4 @@
             })
         }
     }
-</main>
+</main>
\ No newline at end of file
diff --git a/src/UmbracoProject/Views/styleGuidePage.cshtml b/src/UmbracoProject/Views/styleGuidePage.cshtml
index 12fdf62..6cf3b97 100644
--- a/src/UmbracoProject/Views/styleGuidePage.cshtml
+++ b/src/UmbracoProject/Views/styleGuidePage.cshtml
@@ -1,4 +1,4 @@
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.StyleGuidePage>
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.StyleGuidePage>
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels
 @using Umbraco.Cms.Core.Models.Blocks
 @using UmbracoProject.Helpers
@@ -35,4 +35,4 @@
     }
 
     @await Html.PartialAsync("~/Views/Partials/_StyleGuideSectionRows.cshtml", rows)
-</article>
+</article>
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0225af17b30249cb9176b9f35cab9c17.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0225af17b30249cb9176b9f35cab9c17.uda
index 326c5b0..e981046 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0225af17b30249cb9176b9f35cab9c17.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0225af17b30249cb9176b9f35cab9c17.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/0225af17b30249cb9176b9f35cab9c17",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0287f26577e341f4a6393a520a6cf4c0.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0287f26577e341f4a6393a520a6cf4c0.uda
index 183fcc3..13d3ef2 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0287f26577e341f4a6393a520a6cf4c0.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0287f26577e341f4a6393a520a6cf4c0.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/0287f26577e341f4a6393a520a6cf4c0",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0b6a45e744ba430d9da54e46060b9e03.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0b6a45e744ba430d9da54e46060b9e03.uda
index 1cde1b3..c780a2d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0b6a45e744ba430d9da54e46060b9e03.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0b6a45e744ba430d9da54e46060b9e03.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/0b6a45e744ba430d9da54e46060b9e03",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0cc0eba1996042c9bf9b60e150b429ae.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0cc0eba1996042c9bf9b60e150b429ae.uda
index ff69c1d..74b435c 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0cc0eba1996042c9bf9b60e150b429ae.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0cc0eba1996042c9bf9b60e150b429ae.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/0cc0eba1996042c9bf9b60e150b429ae",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e63d883b62b479988c3157f82e83ecc.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e63d883b62b479988c3157f82e83ecc.uda
index 681ff39..8254564 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e63d883b62b479988c3157f82e83ecc.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e63d883b62b479988c3157f82e83ecc.uda
@@ -14,5 +14,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e9794ebf9b54f20a78893acd233a7e4.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e9794ebf9b54f20a78893acd233a7e4.uda
index 773f2a2..ffc5faa 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e9794ebf9b54f20a78893acd233a7e4.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__0e9794ebf9b54f20a78893acd233a7e4.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/0e9794ebf9b54f20a78893acd233a7e4",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1022c77bd4954be8accb77b76f81f5c7.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1022c77bd4954be8accb77b76f81f5c7.uda
index 3b06556..b90ce70 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1022c77bd4954be8accb77b76f81f5c7.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1022c77bd4954be8accb77b76f81f5c7.uda
@@ -13,5 +13,5 @@
   "Udi": "umb://data-type/1022c77bd4954be8accb77b76f81f5c7",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__13037248a9ad4490bd77399fd7913171.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__13037248a9ad4490bd77399fd7913171.uda
index 31cfbf1..69c9eb1 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__13037248a9ad4490bd77399fd7913171.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__13037248a9ad4490bd77399fd7913171.uda
@@ -27,5 +27,5 @@
   "Udi": "umb://data-type/13037248a9ad4490bd77399fd7913171",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1b661f4022424b44b9cb3990ee2b13c0.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1b661f4022424b44b9cb3990ee2b13c0.uda
index 2f08075..d4af6cf 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1b661f4022424b44b9cb3990ee2b13c0.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1b661f4022424b44b9cb3990ee2b13c0.uda
@@ -8,5 +8,5 @@
   "Udi": "umb://data-type/1b661f4022424b44b9cb3990ee2b13c0",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1df9f033e6d4451fb8d2e0cbc50a836f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1df9f033e6d4451fb8d2e0cbc50a836f.uda
index fafa5d6..3b0a738 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1df9f033e6d4451fb8d2e0cbc50a836f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1df9f033e6d4451fb8d2e0cbc50a836f.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type/1df9f033e6d4451fb8d2e0cbc50a836f",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1ea2e01febd84ce18d716b1149e63548.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1ea2e01febd84ce18d716b1149e63548.uda
index ac8b125..e17830f 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1ea2e01febd84ce18d716b1149e63548.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__1ea2e01febd84ce18d716b1149e63548.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/1ea2e01febd84ce18d716b1149e63548",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__20c15a798e4144c2a70efc3933c63cdc.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__20c15a798e4144c2a70efc3933c63cdc.uda
index bc7e999..8552e17 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__20c15a798e4144c2a70efc3933c63cdc.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__20c15a798e4144c2a70efc3933c63cdc.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/20c15a798e4144c2a70efc3933c63cdc",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__215cb418215344299aef8c0f0041191b.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__215cb418215344299aef8c0f0041191b.uda
index 0dbff09..d94a3f3 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__215cb418215344299aef8c0f0041191b.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__215cb418215344299aef8c0f0041191b.uda
@@ -11,5 +11,5 @@
   "Udi": "umb://data-type/215cb418215344299aef8c0f0041191b",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__23f1a6f92a35401491777307b2f34723.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__23f1a6f92a35401491777307b2f34723.uda
index 4ec96fc..2609324 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__23f1a6f92a35401491777307b2f34723.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__23f1a6f92a35401491777307b2f34723.uda
@@ -61,5 +61,5 @@
   "Udi": "umb://data-type/23f1a6f92a35401491777307b2f34723",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2d350d2151dd4c368eee18170985ce91.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2d350d2151dd4c368eee18170985ce91.uda
index 525d6c0..ef0292d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2d350d2151dd4c368eee18170985ce91.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2d350d2151dd4c368eee18170985ce91.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://data-type/2d350d2151dd4c368eee18170985ce91",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2e6d3631066e44b8aec496f09099b2b5.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2e6d3631066e44b8aec496f09099b2b5.uda
index c90001b..01834f6 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2e6d3631066e44b8aec496f09099b2b5.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__2e6d3631066e44b8aec496f09099b2b5.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/2e6d3631066e44b8aec496f09099b2b5",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32854757860e47d2a1a5a66c11857bbd.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32854757860e47d2a1a5a66c11857bbd.uda
index 17c1d10..de3e914 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32854757860e47d2a1a5a66c11857bbd.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32854757860e47d2a1a5a66c11857bbd.uda
@@ -21,5 +21,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3a0156c43b8c4803bdc16871faa83fff.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3a0156c43b8c4803bdc16871faa83fff.uda
index f27708f..d02fcac 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3a0156c43b8c4803bdc16871faa83fff.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3a0156c43b8c4803bdc16871faa83fff.uda
@@ -39,5 +39,5 @@
   "Udi": "umb://data-type/3a0156c43b8c4803bdc16871faa83fff",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3d6a3dc2fe444c368898dbb76f33cd09.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3d6a3dc2fe444c368898dbb76f33cd09.uda
index f2c2910..04d8b17 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3d6a3dc2fe444c368898dbb76f33cd09.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__3d6a3dc2fe444c368898dbb76f33cd09.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/3d6a3dc2fe444c368898dbb76f33cd09",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__4309a3ea0d784329a06cc80b036af19a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__4309a3ea0d784329a06cc80b036af19a.uda
index 2015475..f080c1f 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__4309a3ea0d784329a06cc80b036af19a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__4309a3ea0d784329a06cc80b036af19a.uda
@@ -12,5 +12,5 @@
   "Udi": "umb://data-type/4309a3ea0d784329a06cc80b036af19a",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5046194e4237453ca54715db3a07c4e1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5046194e4237453ca54715db3a07c4e1.uda
index d302499..411c9f8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5046194e4237453ca54715db3a07c4e1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5046194e4237453ca54715db3a07c4e1.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/5046194e4237453ca54715db3a07c4e1",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5d710f95b98842fc9f5833ce973cb511.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5d710f95b98842fc9f5833ce973cb511.uda
index 04e09e3..578fa57 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5d710f95b98842fc9f5833ce973cb511.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5d710f95b98842fc9f5833ce973cb511.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/5d710f95b98842fc9f5833ce973cb511",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5eb57825e15e4fc78e37fca65cdafbde.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5eb57825e15e4fc78e37fca65cdafbde.uda
index 7a32f98..8619204 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5eb57825e15e4fc78e37fca65cdafbde.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5eb57825e15e4fc78e37fca65cdafbde.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://data-type/5eb57825e15e4fc78e37fca65cdafbde",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5f96402a733848429fa3e39347dfb03b.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5f96402a733848429fa3e39347dfb03b.uda
index 84e331e..cf775e6 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5f96402a733848429fa3e39347dfb03b.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__5f96402a733848429fa3e39347dfb03b.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/5f96402a733848429fa3e39347dfb03b",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__70575fe798124396bbe1c81a76db71b5.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__70575fe798124396bbe1c81a76db71b5.uda
index f227fa5..de8ddf6 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__70575fe798124396bbe1c81a76db71b5.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__70575fe798124396bbe1c81a76db71b5.uda
@@ -13,5 +13,5 @@
   "Udi": "umb://data-type/70575fe798124396bbe1c81a76db71b5",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__713d86d059a9442d9f34e64a1675713c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__713d86d059a9442d9f34e64a1675713c.uda
index a93b724..469e75b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__713d86d059a9442d9f34e64a1675713c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__713d86d059a9442d9f34e64a1675713c.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/713d86d059a9442d9f34e64a1675713c",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__72c012d3249346e19dabdbbc177ab1cc.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__72c012d3249346e19dabdbbc177ab1cc.uda
index ee47fad..2324574 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__72c012d3249346e19dabdbbc177ab1cc.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__72c012d3249346e19dabdbbc177ab1cc.uda
@@ -8,5 +8,5 @@
   "Udi": "umb://data-type/72c012d3249346e19dabdbbc177ab1cc",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__765ed8243724414d86dfb0599fe84006.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__765ed8243724414d86dfb0599fe84006.uda
index a2177a5..62d9521 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__765ed8243724414d86dfb0599fe84006.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__765ed8243724414d86dfb0599fe84006.uda
@@ -27,5 +27,5 @@
   "Udi": "umb://data-type/765ed8243724414d86dfb0599fe84006",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__7b21df8f973a4e2d8ee1a77924d9e211.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__7b21df8f973a4e2d8ee1a77924d9e211.uda
index eb7ef3b..ace16b0 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__7b21df8f973a4e2d8ee1a77924d9e211.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__7b21df8f973a4e2d8ee1a77924d9e211.uda
@@ -22,5 +22,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__84c6b44131df4ffeb67e67d5bc3ae65a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__84c6b44131df4ffeb67e67d5bc3ae65a.uda
index 89781f0..07726f7 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__84c6b44131df4ffeb67e67d5bc3ae65a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__84c6b44131df4ffeb67e67d5bc3ae65a.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/84c6b44131df4ffeb67e67d5bc3ae65a",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__856f224e7b134775bd5ab40506f33e04.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__856f224e7b134775bd5ab40506f33e04.uda
index 7cd5829..2bcca15 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__856f224e7b134775bd5ab40506f33e04.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__856f224e7b134775bd5ab40506f33e04.uda
@@ -13,5 +13,5 @@
   "Udi": "umb://data-type/856f224e7b134775bd5ab40506f33e04",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__88e8a05230ee4d44a50759f2cdfc769c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__88e8a05230ee4d44a50759f2cdfc769c.uda
index 6c8259a..75be879 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__88e8a05230ee4d44a50759f2cdfc769c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__88e8a05230ee4d44a50759f2cdfc769c.uda
@@ -11,5 +11,5 @@
   "Udi": "umb://data-type/88e8a05230ee4d44a50759f2cdfc769c",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8e7f995cbd8146279932c40e568ec788.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8e7f995cbd8146279932c40e568ec788.uda
index 2e41c53..71e0d68 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8e7f995cbd8146279932c40e568ec788.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8e7f995cbd8146279932c40e568ec788.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/8e7f995cbd8146279932c40e568ec788",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f1ef1e19de440d3a0726673f631ca64.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f1ef1e19de440d3a0726673f631ca64.uda
index 7a2e8b0..dbc2970 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f1ef1e19de440d3a0726673f631ca64.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f1ef1e19de440d3a0726673f631ca64.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/8f1ef1e19de440d3a0726673f631ca64",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f430dd64e96447e9dc0cb552c8cd1f3.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f430dd64e96447e9dc0cb552c8cd1f3.uda
index 8ef840c..9e00a70 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f430dd64e96447e9dc0cb552c8cd1f3.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8f430dd64e96447e9dc0cb552c8cd1f3.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/8f430dd64e96447e9dc0cb552c8cd1f3",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8fdfe8f374904b9b82f1c0ffb7fdab0b.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8fdfe8f374904b9b82f1c0ffb7fdab0b.uda
index 88c44c8..38f8886 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8fdfe8f374904b9b82f1c0ffb7fdab0b.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__8fdfe8f374904b9b82f1c0ffb7fdab0b.uda
@@ -26,5 +26,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
index 3d1d89c..9eec86b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/92897bc6a5f34ffeae27f2e7e33dda49",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__930861bfe2624eada704f99453565708.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__930861bfe2624eada704f99453565708.uda
index 1dd4bd5..ff6353d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__930861bfe2624eada704f99453565708.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__930861bfe2624eada704f99453565708.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/930861bfe2624eada704f99453565708",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9499b8293e0f4f0daf46a53f2749a5af.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9499b8293e0f4f0daf46a53f2749a5af.uda
index bd141b7..2c1fbef 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9499b8293e0f4f0daf46a53f2749a5af.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9499b8293e0f4f0daf46a53f2749a5af.uda
@@ -11,5 +11,5 @@
   "Udi": "umb://data-type/9499b8293e0f4f0daf46a53f2749a5af",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9a426393b2c84403a85f00b7a51ebe69.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9a426393b2c84403a85f00b7a51ebe69.uda
index 0aba291..0147f7e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9a426393b2c84403a85f00b7a51ebe69.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__9a426393b2c84403a85f00b7a51ebe69.uda
@@ -27,5 +27,5 @@
   "Udi": "umb://data-type/9a426393b2c84403a85f00b7a51ebe69",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a001e2823458461abd10600c93a80de1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a001e2823458461abd10600c93a80de1.uda
index f8609ec..0a63404 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a001e2823458461abd10600c93a80de1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a001e2823458461abd10600c93a80de1.uda
@@ -505,5 +505,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a4807c6e86eb4da8a17e77f1b1c655fa.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a4807c6e86eb4da8a17e77f1b1c655fa.uda
index 48b7562..c0ada8a 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a4807c6e86eb4da8a17e77f1b1c655fa.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a4807c6e86eb4da8a17e77f1b1c655fa.uda
@@ -16,5 +16,5 @@
   "Udi": "umb://data-type/a4807c6e86eb4da8a17e77f1b1c655fa",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a7590e7acad642e18ae9106c717206fd.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a7590e7acad642e18ae9106c717206fd.uda
index 0f36057..da15449 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a7590e7acad642e18ae9106c717206fd.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a7590e7acad642e18ae9106c717206fd.uda
@@ -18,5 +18,5 @@
   "Udi": "umb://data-type/a7590e7acad642e18ae9106c717206fd",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a97cec699b714c308b12ec398860d7e8.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a97cec699b714c308b12ec398860d7e8.uda
index a0d0585..5fb2796 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a97cec699b714c308b12ec398860d7e8.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__a97cec699b714c308b12ec398860d7e8.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/a97cec699b714c308b12ec398860d7e8",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__aa58c9784a99401a96a36f3dd4ca614d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__aa58c9784a99401a96a36f3dd4ca614d.uda
index c34f53f..a01fb5e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__aa58c9784a99401a96a36f3dd4ca614d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__aa58c9784a99401a96a36f3dd4ca614d.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/aa58c9784a99401a96a36f3dd4ca614d",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ad9f0cf2bda245d59ea1a63cfc873fd3.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ad9f0cf2bda245d59ea1a63cfc873fd3.uda
index f610876..00f200d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ad9f0cf2bda245d59ea1a63cfc873fd3.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ad9f0cf2bda245d59ea1a63cfc873fd3.uda
@@ -18,5 +18,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b4e3535a175347e28568602cf8cfee6f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b4e3535a175347e28568602cf8cfee6f.uda
index 68b8505..b49b679 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b4e3535a175347e28568602cf8cfee6f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b4e3535a175347e28568602cf8cfee6f.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type/b4e3535a175347e28568602cf8cfee6f",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b5922818d8d843df88ed4582a24c0fa6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b5922818d8d843df88ed4582a24c0fa6.uda
index d5c6f04..a638136 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b5922818d8d843df88ed4582a24c0fa6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b5922818d8d843df88ed4582a24c0fa6.uda
@@ -109,5 +109,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b6b73142b9c14bf8a16de1c23320b549.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b6b73142b9c14bf8a16de1c23320b549.uda
index 4f5bd22..680dc7b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b6b73142b9c14bf8a16de1c23320b549.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b6b73142b9c14bf8a16de1c23320b549.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/b6b73142b9c14bf8a16de1c23320b549",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b70e6334655e4a92a31e03e0a369e9fc.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b70e6334655e4a92a31e03e0a369e9fc.uda
index f343bab..ebb877b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b70e6334655e4a92a31e03e0a369e9fc.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__b70e6334655e4a92a31e03e0a369e9fc.uda
@@ -19,5 +19,5 @@
   "Udi": "umb://data-type/b70e6334655e4a92a31e03e0a369e9fc",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ba5bdbe6ab3e46a882b32c45f10bc47f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ba5bdbe6ab3e46a882b32c45f10bc47f.uda
index 7bc1634..caa00bc 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ba5bdbe6ab3e46a882b32c45f10bc47f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ba5bdbe6ab3e46a882b32c45f10bc47f.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://data-type/ba5bdbe6ab3e46a882b32c45f10bc47f",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bb5f57c9ce2b4bb9b6974caca783a805.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bb5f57c9ce2b4bb9b6974caca783a805.uda
index d9cc18d..d1d7d89 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bb5f57c9ce2b4bb9b6974caca783a805.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bb5f57c9ce2b4bb9b6974caca783a805.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/bb5f57c9ce2b4bb9b6974caca783a805",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bc1e266cdac44164bf088a1ec6a7143d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bc1e266cdac44164bf088a1ec6a7143d.uda
index 277280e..4042d27 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bc1e266cdac44164bf088a1ec6a7143d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bc1e266cdac44164bf088a1ec6a7143d.uda
@@ -13,5 +13,5 @@
   "Udi": "umb://data-type/bc1e266cdac44164bf088a1ec6a7143d",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bd37253b2f0540e08d386041c9456230.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bd37253b2f0540e08d386041c9456230.uda
index 6e58e54..b62d77a 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bd37253b2f0540e08d386041c9456230.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__bd37253b2f0540e08d386041c9456230.uda
@@ -27,5 +27,5 @@
   "Udi": "umb://data-type/bd37253b2f0540e08d386041c9456230",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c0808dd381334e4b8ce8e2bea84a96a4.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c0808dd381334e4b8ce8e2bea84a96a4.uda
index 90596c7..8fbd68c 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c0808dd381334e4b8ce8e2bea84a96a4.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c0808dd381334e4b8ce8e2bea84a96a4.uda
@@ -39,5 +39,5 @@
   "Udi": "umb://data-type/c0808dd381334e4b8ce8e2bea84a96a4",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c6bac0dd4ab945b18e30e4b619ee5da3.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c6bac0dd4ab945b18e30e4b619ee5da3.uda
index e4244e4..8193763 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c6bac0dd4ab945b18e30e4b619ee5da3.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__c6bac0dd4ab945b18e30e4b619ee5da3.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type/c6bac0dd4ab945b18e30e4b619ee5da3",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ca90c9500aff4e72b976a30b1ac57dad.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ca90c9500aff4e72b976a30b1ac57dad.uda
index a04b9d6..409c5ed 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ca90c9500aff4e72b976a30b1ac57dad.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ca90c9500aff4e72b976a30b1ac57dad.uda
@@ -93,5 +93,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__d4c61dacad0e46c4b2e12020acabd825.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__d4c61dacad0e46c4b2e12020acabd825.uda
index 21ceef9..e8e07d2 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__d4c61dacad0e46c4b2e12020acabd825.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__d4c61dacad0e46c4b2e12020acabd825.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/d4c61dacad0e46c4b2e12020acabd825",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__dec936fbe87b4080abf27466d73f5362.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__dec936fbe87b4080abf27466d73f5362.uda
index 0d93831..a994c08 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__dec936fbe87b4080abf27466d73f5362.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__dec936fbe87b4080abf27466d73f5362.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type/dec936fbe87b4080abf27466d73f5362",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__e4d66c0fb935420081f0025f7256b89a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__e4d66c0fb935420081f0025f7256b89a.uda
index 6795a9a..8b861c8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__e4d66c0fb935420081f0025f7256b89a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__e4d66c0fb935420081f0025f7256b89a.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/e4d66c0fb935420081f0025f7256b89a",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ec08d806a95a477f9ea171c16a4afea7.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ec08d806a95a477f9ea171c16a4afea7.uda
index e5ef674..aab41d0 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ec08d806a95a477f9ea171c16a4afea7.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__ec08d806a95a477f9ea171c16a4afea7.uda
@@ -22,5 +22,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f0bc4bfbb49940d6ba86058885a5178c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f0bc4bfbb49940d6ba86058885a5178c.uda
index a070132..7e45d75 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f0bc4bfbb49940d6ba86058885a5178c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f0bc4bfbb49940d6ba86058885a5178c.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/f0bc4bfbb49940d6ba86058885a5178c",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f38f0ac71d27439c9f3f089cd8825a53.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f38f0ac71d27439c9f3f089cd8825a53.uda
index 9678004..56e3d61 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f38f0ac71d27439c9f3f089cd8825a53.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f38f0ac71d27439c9f3f089cd8825a53.uda
@@ -9,5 +9,5 @@
   "Udi": "umb://data-type/f38f0ac71d27439c9f3f089cd8825a53",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f8d7b987745c471f8f34a47f00838f06.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f8d7b987745c471f8f34a47f00838f06.uda
index c1354e4..e78c731 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f8d7b987745c471f8f34a47f00838f06.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__f8d7b987745c471f8f34a47f00838f06.uda
@@ -25,5 +25,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fbaf13a8403641f293a3974f678c312a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fbaf13a8403641f293a3974f678c312a.uda
index 57a8fcc..e438fad 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fbaf13a8403641f293a3974f678c312a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fbaf13a8403641f293a3974f678c312a.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type/fbaf13a8403641f293a3974f678c312a",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fccec1e1005748cab9bf184e00758a58.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fccec1e1005748cab9bf184e00758a58.uda
index 8c2a2d9..1e894c9 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fccec1e1005748cab9bf184e00758a58.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fccec1e1005748cab9bf184e00758a58.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://data-type/fccec1e1005748cab9bf184e00758a58",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fd1e0da556064862b6795d0cf3a52a59.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fd1e0da556064862b6795d0cf3a52a59.uda
index edaf973..06540aa 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fd1e0da556064862b6795d0cf3a52a59.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__fd1e0da556064862b6795d0cf3a52a59.uda
@@ -7,5 +7,5 @@
   "Udi": "umb://data-type/fd1e0da556064862b6795d0cf3a52a59",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__03c19707d84540608db45a17698117ef.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__03c19707d84540608db45a17698117ef.uda
index a935ed6..17d81fd 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__03c19707d84540608db45a17698117ef.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__03c19707d84540608db45a17698117ef.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__1e21e1edb4ef4722a0a304ac014239c3.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__1e21e1edb4ef4722a0a304ac014239c3.uda
index ae4f09b..38ebd16 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__1e21e1edb4ef4722a0a304ac014239c3.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__1e21e1edb4ef4722a0a304ac014239c3.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2683959fe16b47d0ae981c8bf5894af6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2683959fe16b47d0ae981c8bf5894af6.uda
index 15ad6f9..087019b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2683959fe16b47d0ae981c8bf5894af6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2683959fe16b47d0ae981c8bf5894af6.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2aeeb513ea004787be739d0c4a134246.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2aeeb513ea004787be739d0c4a134246.uda
index 340a8a5..a2dae48 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2aeeb513ea004787be739d0c4a134246.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2aeeb513ea004787be739d0c4a134246.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2b8182a1dd004fbd991d4f021b8f0710.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2b8182a1dd004fbd991d4f021b8f0710.uda
index 048679c..74fe1a3 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2b8182a1dd004fbd991d4f021b8f0710.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__2b8182a1dd004fbd991d4f021b8f0710.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__30e9e2c8385449ab9bc2bd4947bb405e.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__30e9e2c8385449ab9bc2bd4947bb405e.uda
index e3642cc..ed89f7a 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__30e9e2c8385449ab9bc2bd4947bb405e.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__30e9e2c8385449ab9bc2bd4947bb405e.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__319dc106c9d443f18baa82996e6edea2.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__319dc106c9d443f18baa82996e6edea2.uda
index 67c6b9b..d6f32ff 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__319dc106c9d443f18baa82996e6edea2.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__319dc106c9d443f18baa82996e6edea2.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3699b66f963e4b24878583710c2755e1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3699b66f963e4b24878583710c2755e1.uda
index 4080dc6..89f62f4 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3699b66f963e4b24878583710c2755e1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3699b66f963e4b24878583710c2755e1.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3bc3234967184b05b886fff878380df5.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3bc3234967184b05b886fff878380df5.uda
index 0dbf8fe..296e9a0 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3bc3234967184b05b886fff878380df5.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__3bc3234967184b05b886fff878380df5.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__52855e4c75c1456d84ee7f7956d2b604.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__52855e4c75c1456d84ee7f7956d2b604.uda
index 16eb83f..8d6fa00 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__52855e4c75c1456d84ee7f7956d2b604.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__52855e4c75c1456d84ee7f7956d2b604.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5292acd7095243b99744e72216e9210f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5292acd7095243b99744e72216e9210f.uda
index 828c75a..869528d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5292acd7095243b99744e72216e9210f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5292acd7095243b99744e72216e9210f.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5a3671a2cc2944208e10ddf3b14b095a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5a3671a2cc2944208e10ddf3b14b095a.uda
index ddf68a2..7f8924b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5a3671a2cc2944208e10ddf3b14b095a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5a3671a2cc2944208e10ddf3b14b095a.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5d3d71d7a00246af9763bd71159af986.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5d3d71d7a00246af9763bd71159af986.uda
index 903f576..27a500e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5d3d71d7a00246af9763bd71159af986.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__5d3d71d7a00246af9763bd71159af986.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__62cbe7af2ff14584b2e26f50cf11e116.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__62cbe7af2ff14584b2e26f50cf11e116.uda
index e25b0d4..499930e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__62cbe7af2ff14584b2e26f50cf11e116.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__62cbe7af2ff14584b2e26f50cf11e116.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7198b952d2b643c2adfe4978b3a7a9f9.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7198b952d2b643c2adfe4978b3a7a9f9.uda
index 137311f..2fc7e72 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7198b952d2b643c2adfe4978b3a7a9f9.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7198b952d2b643c2adfe4978b3a7a9f9.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__793e93e94b984fec816e2dcb4a606614.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__793e93e94b984fec816e2dcb4a606614.uda
index a90068d..9722cb9 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__793e93e94b984fec816e2dcb4a606614.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__793e93e94b984fec816e2dcb4a606614.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7cef2c8b50884f49b67ef2b5fe0f35eb.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7cef2c8b50884f49b67ef2b5fe0f35eb.uda
index 5db396a..03c69f5 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7cef2c8b50884f49b67ef2b5fe0f35eb.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7cef2c8b50884f49b67ef2b5fe0f35eb.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7e02a26c5a424e3592fd3b4ab480d7d5.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7e02a26c5a424e3592fd3b4ab480d7d5.uda
index 191b390..197a98a 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7e02a26c5a424e3592fd3b4ab480d7d5.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__7e02a26c5a424e3592fd3b4ab480d7d5.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__89160a682e2c4fa1a715614a8187b5d9.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__89160a682e2c4fa1a715614a8187b5d9.uda
index 56fb0f2..52b1620 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__89160a682e2c4fa1a715614a8187b5d9.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__89160a682e2c4fa1a715614a8187b5d9.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__92227cb8914345f1926b8c54d091e642.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__92227cb8914345f1926b8c54d091e642.uda
index d2576eb..5a8034e 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__92227cb8914345f1926b8c54d091e642.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__92227cb8914345f1926b8c54d091e642.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__967b655937ae48faaf464e52af5cbcac.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__967b655937ae48faaf464e52af5cbcac.uda
index 484ed19..6bb8dce 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__967b655937ae48faaf464e52af5cbcac.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__967b655937ae48faaf464e52af5cbcac.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__9d108073cb85427d80a259eeb65a1147.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__9d108073cb85427d80a259eeb65a1147.uda
index 40eb4d5..86d3903 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__9d108073cb85427d80a259eeb65a1147.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__9d108073cb85427d80a259eeb65a1147.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__c0e753839b594272be1b7bd0b29078c0.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__c0e753839b594272be1b7bd0b29078c0.uda
index 55898e4..d56c8de 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__c0e753839b594272be1b7bd0b29078c0.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__c0e753839b594272be1b7bd0b29078c0.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__d31aca27aff24cc78a7b3dbea5d27ba9.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__d31aca27aff24cc78a7b3dbea5d27ba9.uda
index e504f0c..b616470 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__d31aca27aff24cc78a7b3dbea5d27ba9.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__d31aca27aff24cc78a7b3dbea5d27ba9.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__db875bf85b2248abba696682c3b7e45a.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__db875bf85b2248abba696682c3b7e45a.uda
index 06250fd..37973d5 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__db875bf85b2248abba696682c3b7e45a.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__db875bf85b2248abba696682c3b7e45a.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__e0c08e95fb8c4adabf3cf5d722cecd02.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__e0c08e95fb8c4adabf3cf5d722cecd02.uda
index a1f6e90..c8a1d11 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__e0c08e95fb8c4adabf3cf5d722cecd02.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__e0c08e95fb8c4adabf3cf5d722cecd02.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f3edbc573e5d49efa7fd0e9de83208ec.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f3edbc573e5d49efa7fd0e9de83208ec.uda
index e662ecc..9073631 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f3edbc573e5d49efa7fd0e9de83208ec.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f3edbc573e5d49efa7fd0e9de83208ec.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f55e2b7d8df445d8826c99ce2a11f537.uda b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f55e2b7d8df445d8826c99ce2a11f537.uda
index e53464a..7a76f1c 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f55e2b7d8df445d8826c99ce2a11f537.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/dictionary-item__f55e2b7d8df445d8826c99ce2a11f537.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DictionaryItemArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__1645b9b1459b40e790a5ea194afda61d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__1645b9b1459b40e790a5ea194afda61d.uda
index 54fc8a6..3d9da8d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__1645b9b1459b40e790a5ea194afda61d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__1645b9b1459b40e790a5ea194afda61d.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__3503b89f28194e4186d7d17dcc5b4212.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__3503b89f28194e4186d7d17dcc5b4212.uda
index eb5b5c4..931b6da 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__3503b89f28194e4186d7d17dcc5b4212.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__3503b89f28194e4186d7d17dcc5b4212.uda
@@ -4,5 +4,5 @@
   "Udi": "umb://document-type-container/3503b89f28194e4186d7d17dcc5b4212",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__5dde5b35b5f94d61aaf1158368a1b0fb.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__5dde5b35b5f94d61aaf1158368a1b0fb.uda
index 4d2a26f..0b052e8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__5dde5b35b5f94d61aaf1158368a1b0fb.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__5dde5b35b5f94d61aaf1158368a1b0fb.uda
@@ -4,5 +4,5 @@
   "Udi": "umb://document-type-container/5dde5b35b5f94d61aaf1158368a1b0fb",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a2c7196096784b569828c1d8f8f7df40.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a2c7196096784b569828c1d8f8f7df40.uda
index 516b841..e01b2e1 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a2c7196096784b569828c1d8f8f7df40.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a2c7196096784b569828c1d8f8f7df40.uda
@@ -4,5 +4,5 @@
   "Udi": "umb://document-type-container/a2c7196096784b569828c1d8f8f7df40",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a3274987179946d6885c551ba2986c90.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a3274987179946d6885c551ba2986c90.uda
index 5079fc7..dd1162d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a3274987179946d6885c551ba2986c90.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__a3274987179946d6885c551ba2986c90.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__e342fe7205e3452fbb396c68234eb664.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__e342fe7205e3452fbb396c68234eb664.uda
index f8908bc..63dc4ec 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__e342fe7205e3452fbb396c68234eb664.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__e342fe7205e3452fbb396c68234eb664.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__f42b337f5693436491df1240411bc43f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__f42b337f5693436491df1240411bc43f.uda
index e62b4ca..92ca189 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__f42b337f5693436491df1240411bc43f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type-container__f42b337f5693436491df1240411bc43f.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a0eec50c54ce47d397f1c01843887567.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a0eec50c54ce47d397f1c01843887567.uda
index 2431bc4..59cafe2 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a0eec50c54ce47d397f1c01843887567.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a0eec50c54ce47d397f1c01843887567.uda
@@ -30,5 +30,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e33a8f18ae714a7dbc2c76ceb9fd9e97.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e33a8f18ae714a7dbc2c76ceb9fd9e97.uda
index 034de19..277aaf0 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e33a8f18ae714a7dbc2c76ceb9fd9e97.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__e33a8f18ae714a7dbc2c76ceb9fd9e97.uda
@@ -21,5 +21,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/member-group__5174a9bf70e74304a714d484e47a50f6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/member-group__5174a9bf70e74304a714d484e47a50f6.uda
index 1675531..63e8696 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/member-group__5174a9bf70e74304a714d484e47a50f6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/member-group__5174a9bf70e74304a714d484e47a50f6.uda
@@ -3,5 +3,5 @@
   "Udi": "umb://member-group/5174a9bf70e74304a714d484e47a50f6",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.MemberGroupArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__019cf7abd0cc72a8b1b100ac020bd309.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__019cf7abd0cc72a8b1b100ac020bd309.uda
index d795ceb..217c848 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__019cf7abd0cc72a8b1b100ac020bd309.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__019cf7abd0cc72a8b1b100ac020bd309.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__23e621c7df614f4f9ef72daa193c63b8.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__23e621c7df614f4f9ef72daa193c63b8.uda
index 4ad18a4..9d38a2d 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__23e621c7df614f4f9ef72daa193c63b8.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__23e621c7df614f4f9ef72daa193c63b8.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__26611ae115484ba5a6960cabed50057c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__26611ae115484ba5a6960cabed50057c.uda
index 746ba3f..e398d78 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__26611ae115484ba5a6960cabed50057c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__26611ae115484ba5a6960cabed50057c.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__72f84ac5d0434035937f2f0a99d6afbe.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__72f84ac5d0434035937f2f0a99d6afbe.uda
index d1286eb..9987e77 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__72f84ac5d0434035937f2f0a99d6afbe.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__72f84ac5d0434035937f2f0a99d6afbe.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__898d5e2300c24d548dd2ebcd39b9ad1f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__898d5e2300c24d548dd2ebcd39b9ad1f.uda
index 7d4981e..b9eeea7 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__898d5e2300c24d548dd2ebcd39b9ad1f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__898d5e2300c24d548dd2ebcd39b9ad1f.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__907d26a5c9294c69bb1d0d0d5342d6f6.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__907d26a5c9294c69bb1d0d0d5342d6f6.uda
index 4b0b480..4c45e78 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__907d26a5c9294c69bb1d0d0d5342d6f6.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__907d26a5c9294c69bb1d0d0d5342d6f6.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__9fd73dbb5133435c94bb334eb887cf9c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__9fd73dbb5133435c94bb334eb887cf9c.uda
index 2c88837..b246be5 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__9fd73dbb5133435c94bb334eb887cf9c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__9fd73dbb5133435c94bb334eb887cf9c.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__a4f8c1b27d3e4f9a8b6c1e2f3a4b5c6d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__a4f8c1b27d3e4f9a8b6c1e2f3a4b5c6d.uda
index 1058594..d46a936 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__a4f8c1b27d3e4f9a8b6c1e2f3a4b5c6d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__a4f8c1b27d3e4f9a8b6c1e2f3a4b5c6d.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__b0cdefabb7e648d3a1771242d581149d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__b0cdefabb7e648d3a1771242d581149d.uda
index 4b15ed4..7e8c149 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__b0cdefabb7e648d3a1771242d581149d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__b0cdefabb7e648d3a1771242d581149d.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__c25c56ee7808462a9dc80258d091e1f4.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__c25c56ee7808462a9dc80258d091e1f4.uda
index 3564b88..65f8a60 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__c25c56ee7808462a9dc80258d091e1f4.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__c25c56ee7808462a9dc80258d091e1f4.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__c604acfe4b214e4681f4b2b34d850fd1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__c604acfe4b214e4681f4b2b34d850fd1.uda
index 494095b..8de5cfa 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__c604acfe4b214e4681f4b2b34d850fd1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__c604acfe4b214e4681f4b2b34d850fd1.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__c83d63947bef438c929d892b42066b37.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__c83d63947bef438c929d892b42066b37.uda
index aa8dc89..fee11f8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__c83d63947bef438c929d892b42066b37.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__c83d63947bef438c929d892b42066b37.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__d82c35eed4df44bb87471c7d1ae35ee1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__d82c35eed4df44bb87471c7d1ae35ee1.uda
index 29b3a1b..6ac6792 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__d82c35eed4df44bb87471c7d1ae35ee1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__d82c35eed4df44bb87471c7d1ae35ee1.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__e1eefce386e14a7e899939ab890a6c73.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__e1eefce386e14a7e899939ab890a6c73.uda
index 09dc0db..8cfc369 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__e1eefce386e14a7e899939ab890a6c73.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__e1eefce386e14a7e899939ab890a6c73.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__e265abdbce7c4646b472ca922b313b18.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__e265abdbce7c4646b472ca922b313b18.uda
index 97a649d..f4c49c8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__e265abdbce7c4646b472ca922b313b18.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__e265abdbce7c4646b472ca922b313b18.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__ee0257b99fb04e90bc99f8ab75c6100e.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__ee0257b99fb04e90bc99f8ab75c6100e.uda
index bea117a..6e48520 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__ee0257b99fb04e90bc99f8ab75c6100e.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__ee0257b99fb04e90bc99f8ab75c6100e.uda
@@ -11,5 +11,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.1"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__ef27c2a5e5e44a29a35f80f3c15ca4d2.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__ef27c2a5e5e44a29a35f80f3c15ca4d2.uda
index 7bc2b69..a09a413 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__ef27c2a5e5e44a29a35f80f3c15ca4d2.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__ef27c2a5e5e44a29a35f80f3c15ca4d2.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__79f50bf8aafb4ae88835e741c3371a43.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__79f50bf8aafb4ae88835e741c3371a43.uda
index 57021ea..a4facbc 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__79f50bf8aafb4ae88835e741c3371a43.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__79f50bf8aafb4ae88835e741c3371a43.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://umbraco-ai-connection/79f50bf8aafb4ae88835e741c3371a43",
   "Dependencies": [],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIConnectionArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__a3599365553d4b958bcae0422b3c8e1c.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__a3599365553d4b958bcae0422b3c8e1c.uda
index c328191..8a38671 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__a3599365553d4b958bcae0422b3c8e1c.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-connection__a3599365553d4b958bcae0422b3c8e1c.uda
@@ -10,5 +10,5 @@
   "Udi": "umb://umbraco-ai-connection/a3599365553d4b958bcae0422b3c8e1c",
   "Dependencies": [],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIConnectionArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
index 139b765..22f113a 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__1d2c0e55ffab48db92a3d6d589af4f80.uda
@@ -3,7 +3,7 @@
   "Alias": "site-content-guidelines",
   "Resources": [
     {
-      "Id": "91f21274-1d64-420b-9339-27a7f065b96b",
+      "Id": "f3b9804d-1fb9-49bf-bb60-67b2f55b1ad8",
       "ResourceTypeId": "brand-voice",
       "Name": "Brand Voice",
       "Description": null,
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
index c671608..574c8c5 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__d11be0caf8fb4d8bb3bf1ef37ff9e624.uda
@@ -3,7 +3,7 @@
   "Alias": "how-to-guide-style",
   "Resources": [
     {
-      "Id": "4cc07103-8f60-40e0-a447-91fdc6ab409b",
+      "Id": "e18dc67b-c751-4142-86f9-f75e7c6bc8ad",
       "ResourceTypeId": "text",
       "Name": "Structural Template",
       "Description": "Section-by-section structure every generated guide must follow.",
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__8bc9aabc4fff4d9681a58a06f554893b.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__8bc9aabc4fff4d9681a58a06f554893b.uda
index 618463e..a40dcd6 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__8bc9aabc4fff4d9681a58a06f554893b.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__8bc9aabc4fff4d9681a58a06f554893b.uda
@@ -13,5 +13,5 @@
     }
   ],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIProfileArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__b2babe4c84b341439cc889c68949ed83.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__b2babe4c84b341439cc889c68949ed83.uda
index 18e9fc4..79654fb 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__b2babe4c84b341439cc889c68949ed83.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__b2babe4c84b341439cc889c68949ed83.uda
@@ -13,5 +13,5 @@
     }
   ],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIProfileArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__bfcc69a87b464598abc27ac55e3f3c13.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__bfcc69a87b464598abc27ac55e3f3c13.uda
index 82d4c2d..810c08b 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__bfcc69a87b464598abc27ac55e3f3c13.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__bfcc69a87b464598abc27ac55e3f3c13.uda
@@ -14,5 +14,5 @@
     }
   ],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIProfileArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__debdb0b149df43f69f0ad6d56fea514e.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__debdb0b149df43f69f0ad6d56fea514e.uda
index c81ff7b..f11f6b0 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__debdb0b149df43f69f0ad6d56fea514e.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-profile__debdb0b149df43f69f0ad6d56fea514e.uda
@@ -13,5 +13,5 @@
     }
   ],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AIProfileArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__3733171b2cba4f9f99a0dac038db5d5f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__3733171b2cba4f9f99a0dac038db5d5f.uda
index c92d92f..7d66458 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__3733171b2cba4f9f99a0dac038db5d5f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__3733171b2cba4f9f99a0dac038db5d5f.uda
@@ -5,6 +5,7 @@
   "Instructions": "Generate alt text for the image {{ image:umbracoFile }}. The text generated should follow best practices for web accessibility, so that people using assistive technology can access the content. You understand the rules about when alt text should and should not be applied, and do not suggest alt text for decorative images or for SEO-only reasons. Descriptions should always be less that 120 characters. ",
   "ProfileUdi": "umb://umbraco-ai-profile/8bc9aabc4fff4d9681a58a06f554893b",
   "ContextIds": [],
+  "GuardrailIds": [],
   "Tags": [],
   "IsActive": true,
   "IncludeEntityContext": true,
@@ -30,5 +31,5 @@
     }
   ],
   "__type": "Umbraco.AI.Prompt.Deploy,Umbraco.AI.Prompt.Deploy.Artifacts.AIPromptArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__ed8aef69aa27438aa2a9872dfe0e4dfe.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__ed8aef69aa27438aa2a9872dfe0e4dfe.uda
index 9a989b3..3e48aa2 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__ed8aef69aa27438aa2a9872dfe0e4dfe.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-prompt__ed8aef69aa27438aa2a9872dfe0e4dfe.uda
@@ -7,6 +7,7 @@
   "ContextIds": [
     "1d2c0e55-ffab-48db-92a3-d6d589af4f80"
   ],
+  "GuardrailIds": [],
   "Tags": [],
   "IsActive": true,
   "IncludeEntityContext": true,
@@ -33,5 +34,5 @@
     }
   ],
   "__type": "Umbraco.AI.Prompt.Deploy,Umbraco.AI.Prompt.Deploy.Artifacts.AIPromptArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-settings__672bf83c97e04d049d3323fc2e5ebe42.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-settings__672bf83c97e04d049d3323fc2e5ebe42.uda
index cdd9018..eb38381 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-settings__672bf83c97e04d049d3323fc2e5ebe42.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-settings__672bf83c97e04d049d3323fc2e5ebe42.uda
@@ -12,5 +12,5 @@
     }
   ],
   "__type": "Umbraco.AI.Deploy,Umbraco.AI.Deploy.Artifacts.AISettingsArtifact",
-  "__version": "17.0.2"
+  "__version": "17.1.0"
 }
\ No newline at end of file
