diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32deaf6058fe4ce4a21cf6d88f206c75.uda b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32deaf6058fe4ce4a21cf6d88f206c75.uda
index c01f1c5..f8e65fe 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32deaf6058fe4ce4a21cf6d88f206c75.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/data-type__32deaf6058fe4ce4a21cf6d88f206c75.uda
@@ -768,5 +768,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__10554847fbd94a12826c7c4663b841e9.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__10554847fbd94a12826c7c4663b841e9.uda
index 4a33ac3..a40d5f3 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__10554847fbd94a12826c7c4663b841e9.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__10554847fbd94a12826c7c4663b841e9.uda
@@ -40,5 +40,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__1c46d84542214d2e9e1dcfa5b47cbee1.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__1c46d84542214d2e9e1dcfa5b47cbee1.uda
index 848c2f1..6c31aa8 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__1c46d84542214d2e9e1dcfa5b47cbee1.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__1c46d84542214d2e9e1dcfa5b47cbee1.uda
@@ -57,5 +57,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a95360e8ff0440b18f467aa4b5983096.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a95360e8ff0440b18f467aa4b5983096.uda
index eda8955..e071b06 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a95360e8ff0440b18f467aa4b5983096.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__a95360e8ff0440b18f467aa4b5983096.uda
@@ -144,5 +144,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__d977a895b8904bed8742a89a1926f55d.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__d977a895b8904bed8742a89a1926f55d.uda
index dce246e..07ff01f 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__d977a895b8904bed8742a89a1926f55d.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__d977a895b8904bed8742a89a1926f55d.uda
@@ -56,5 +56,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__cdd72ffcacfa41c6ae84e11446425b7f.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__cdd72ffcacfa41c6ae84e11446425b7f.uda
index 31e74cd..2ed62b5 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/template__cdd72ffcacfa41c6ae84e11446425b7f.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__cdd72ffcacfa41c6ae84e11446425b7f.uda
@@ -17,5 +17,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "17.2.0"
+  "__version": "17.2.1"
 }
\ No newline at end of file
