diff --git a/src/UmbracoProject/UmbracoProject.csproj b/src/UmbracoProject/UmbracoProject.csproj
index 50b69b7..3b9dd19 100644
--- a/src/UmbracoProject/UmbracoProject.csproj
+++ b/src/UmbracoProject/UmbracoProject.csproj
@@ -73,7 +73,7 @@
     <PackageReference Include="Umbraco.AI.Prompt" Version="17.0.1" />
     <PackageReference Include="Umbraco.AI.Prompt.Deploy" Version="17.0.0" />
     <PackageReference Include="Umbraco.AI.Search" Version="17.0.0" />
-    <PackageReference Include="Umbraco.Cloud.Cms" Version="17.2.0" />
+    <PackageReference Include="Umbraco.Cloud.Cms" Version="17.2.1" />
     <PackageReference Include="Umbraco.Cloud.StorageProviders.AzureBlob" Version="17.1.0" />
     <PackageReference Include="Umbraco.Cms" Version="17.5.3" />
     <PackageReference Include="Umbraco.Cms.DevelopmentMode.Backoffice" Version="17.5.3" />
