diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__9471c1a00aee463bb4e6f47e89ad102b.uda b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__9471c1a00aee463bb4e6f47e89ad102b.uda
index c9b2598..5c5b679 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__9471c1a00aee463bb4e6f47e89ad102b.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/umbraco-ai-context__9471c1a00aee463bb4e6f47e89ad102b.uda
@@ -3,7 +3,7 @@
   "Alias": "seo-aeo-guidance",
   "Resources": [
     {
-      "Id": "0ab932d8-9b4f-4d80-bb97-b5acbd1f29d6",
+      "Id": "6c4946c8-b4ba-4f58-b66a-80ca246c4137",
       "ResourceTypeId": "text",
       "Name": "Text",
       "Description": "Shared best practices to optimized for SEO and AEO",
