diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b871f83c23954894be0f5422c1a71e48.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b871f83c23954894be0f5422c1a71e48.uda
index 2649c14..d9013d3 100644
--- a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b871f83c23954894be0f5422c1a71e48.uda
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__b871f83c23954894be0f5422c1a71e48.uda
@@ -81,5 +81,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "17.1.0"
+  "__version": "17.2.0"
 }
\ No newline at end of file
